### DISCUSSION TITLE
sbrscrape is not pulling historical odds

### DISCUSSION THREAD

**Discussion by ewalsh3963** [kyleskom/NBA-Machine-Learning-Sports-Betting#286] [General]
Hello

I was trying to run the Get_Odds_Data.py script when I encountered Scoreboard from sbrscrape was not working how I thought it would. When I set a date to pull the odds for 2022-10-19 it actually shows the current odds available on the site. Has anyone using this repo encountered this problem?

https://www.sportsbookreview.com/betting-odds/nba-basketball/

>>> sb = Scoreboard(date='2022-10-19')
>>> sb.games
[{'date': '2023-10-05T16:00:00+00:00', 'status': '12:00 ET', 'home_team': 'Minnesota Timberwolves', 'home_team_loc': 'Minnesota', 'home_team_abbr': 'MIN', 'home_team_rank': -1, 'away_team': 'Dallas Mavericks', 'away_team_loc': 'Dallas', 'away_team_abbr': 'DAL', 'away_team_rank': -1, 'home_score': 0, 'away_score': 0, 'home_spread': {}, 'home_spread_odds': {}, 'away_spread': {}, 'away_spread_odds': {}, 'under_odds': {}, 'over_odds': {}, 'total': {}, 'home_ml': {}, 'away_ml': {}}]

**Comment by kyleskom** [comment]
This has been fixed.
